libensemble/gen_funcs/aposmm_localopt_support.py

```diff
@@ -202,6 +195,8 @@ def run_local_nlopt(user_specs, comm_queue, x0, f0, child_can_read, parent_can_r
     Runs an NLopt local optimization run starting at ``x0``, governed by the
     parameters in ``user_specs``.
     """
+    import nlopt
```

Comment:
Can we ensure that moving this import to so late doesn't cause issues when trying to run with (say) 64 workers?

Reply:
There was no issue in my tests.